frontend/public/components/marketplace/marketplace-subscribe.tsx

```diff
@@ -57,15 +57,17 @@ export const MarketplaceSubscribeForm = withFormState((props: MarketplaceSubscri
       ? packageName
       : _.uniq(catalogSourceConfig.spec.packages.split(',').concat([packageName])).join(',');
 
+    const operatorGroupNamespace = props.operatorGroup.data.find(og => og.metadata.name === props.formState().target).metadata.namespace;
+
     const newCatalogSourceConfig = {
       apiVersion: `${CatalogSourceConfigModel.apiGroup}/${CatalogSourceConfigModel.apiVersion}`,
       kind: CatalogSourceConfigModel.kind,
       metadata: {
         name: MARKETPLACE_CSC_NAME,
-        namespace: 'openshift-operators',
+        namespace: 'openshift-marketplace',
       },
       spec: {
-        targetNamespace: 'openshift-operators',
+        targetNamespace: `${operatorGroupNamespace}`,
```

Comment:
No need to use backticks, here and below.

```suggestion
        targetNamespace: operatorGroupNamespace,
```